src/styles/components/_new-release-banner.scss

```diff
@@ -0,0 +1,78 @@
+@import '../abstracts/_abstracts-dir';
+
+.new-release-banner {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: rem(162px);
+
+  background: {
+    image: url('/images/new-release-banner.png');
+    position: center;
+    repeat: no-repeat;
+    size: cover;
+  }
+
+  @include respond-above('md') {
+    height: rem(300px);
+  }
+
+  @include respond-above('lg') {
+    height: rem(388px);
+  }
+
+  @include respond-above('xl') {
+    height: rem(474px);
+  }
+
+  &__header {
```

Comment:
Would be better to add the side `paddings` at `new-release-banner` and add `margin-bottom` at `header`.